AGPs/AGP-1.md

```diff
@@ -65,6 +65,30 @@ Proposals made to the Finance track must affect the movement of assets held by t
   * GPL
   * AGPL
   * MIT
+  
+_Finance track budget_
+
+The total amount funded by Finance track AGPs shall not exceed 5% of the market value of the Association / Aragon Network treasury or 250,000 DAI, whichever is greater, in a given quarterly Aragon Network vote cycle. Recurring transfers approved in previous vote cycles shall be deducted from this budget as described below. The market value of the treasury, and spending relative to that, shall be measured in DAI at the time an attempt is made to process an approved Finance track proposal as described below.
+
+For any Aragon Network vote ballot, approved Finance track proposals will be processed in the following order:
+
+- First, process proposals that reduce or eliminate a previously approved recurring transfer.
+
+- Then, attempt to debit any ongoing recurring transfers from this quarter’s budget, except ongoing recurring transfers whose amount has been increased in the most recent vote cycle.
+
+- Finally, attempt to process approved Finance track AGPs, that either make new transfers or increase previously approved recurring transfers, in descending order based on absolute approval percentage. Each processed proposal will be debited against this quarter’s remaining budget, and if there is not enough budget available for a proposal when it attempts to be processed then processing will fail and the approved transfer will not be made. An attempt will then be made to process the next approved proposal until there are no more remaining approved proposals to process.
```

Comment:
If a Finance AGP cannot be processed, I assume that it would need to be put up for a vote again as even if in a future ANV there's space in the budget it won't process proposals that couldn't be processed before. I think this makes sense, but I think it should be made explicit